### ISSUE TITLE
Issue when building URL : AddToExecutionQueue ?

### ISSUE THREAD

**Issue by R-Loic** [cerberustesting/cerberus-jenkins-plugin#4]
Hello,
I have the following issue in jenkins : 

_Started by user Loic Roose (external)
Building in workspace D:\jenkins\workspace\TestCerberus2
[INFO] [2017-07-31 09:45:55.4555] Launch campaign RevisionSimulator on http://cerberus.midaseurope.net with tag toto_20170731094555284 
[WARNING] [2017-07-31 09:45:55.4555] browserVersion is empty, platform is empty, 
[ERROR] [2017-07-31 09:45:55.4555] Error message when trying to add a new exectution in queue : Internal Server Error
[ERROR] [2017-07-31 09:45:55.4555] Fail to add campaign RevisionSimulator in cerberus queue
[ERROR] [2017-07-31 09:45:55.4555] UNSTABLE
Build step 'Execute Cerberus Campaign' changed build result to UNSTABLE
Finished: UNSTABLE_

For this job, I have set the same parameters in jenkins as the following URL : 

https://cerberus/AddToExecutionQueue?OutputFormat=json&Robot=HubSelenium&Environment=INT&ss_ip=XXXXXX&ss_p=XXXX&Browser=firefox&BrowserVersion=&Platform=&Tag=TestURLForJenkins&Screenshot=1&Verbose=1&timeout=5000&Synchroneous=Y&PageSource=1&SeleniumLog=1&retries=0&manualExecution=N&SelectedCampaign=RevisionSimulator

The URL runs the campaign as expected but not the job. Could you please add to the output info the URL build by jenkins job ? It will be much easier to debug.

Thank you !

**Comment by nicodeur** [comment]
Hi @R-Loic ,

I had displayed url before, but I prefer hide it to hide the complexity and avoid to launch several times a campaign with the same tag.

To get arround the problem, I added the message of style : 
```
[WARNING] [2017-07-31 09:45:55.4555] browserVersion is empty, platform is empty
```

Job verify each parameter one by one. Perhaps I forgot a rules for a parameter.
I can also display all parameters lines by lines ... but it's just display that you enter on global and job settings.

What do you think about it ?

To  resolve quickly your problem, can you send me a print screen of your settings (global and job settings) ?

**Comment by nicodeur** [comment]
Hi,
Problem is due to Cerberus. Fixed on version 1.14.0. I added some text on console and settings to be more userfriendly.

Regards,

Nicolas
